Whiskey/en-US/about_Whiskey_DotNetTest_Task.help.txt

```diff
@@ -0,0 +1,71 @@
+TOPIC
+    about_Whiskey_DotNetTest_Task
+
+SUMMARY
+    Runs unit tests for a .NET Core project.
+
+DESCRIPTION
+    The `DotNetTest` tasks runs the `dotnet test` command to run unit tests for a .NET Core project. Pass a list of solution files or .NET Core project files to the `Path` property. If no files are provided to `Path`, then the .NET Core SDK will search for any solution or project files in the working directory. If the `dotnet test` command returns a non-zero exit code the build will fail.
+
+    The task will automatically use the following options with the `dotnet test` command and are not modifiable:
+
+        * `--configuration=(Debug|Release)`: `Debug` when build run by a developer and `Release` when build run by a build server.
+        * `--no-build`: Skip building the project prior to testing. Use Whiskey's `DotNetBuild` task to build the test project.
+        * `--results-directory=$(WHISKEY_OUTPUT_DIRECTORY)`: The default Whiskey build output directory.

```

Comment:
Add some text about the output, e.g. "By default, test results are not saved to a file. Use the `Logger` property to specify the logger URI/friendly name (e.g. `trx`). The results will be saved to a file in Whiskey's .output directory."